docs/02-StewardForValidators.md

```diff
@@ -0,0 +1,36 @@
+# Validators Instructions for Setting Up Steward
+
+Steward is an application intended for developers and validators on the Sommelier network.
+
+It can run as a server for forwarding cellar contract calls from Strategy Providers through the Sommelier chain, or it can run in test mode to directly interact with Ethereum contracts as a single signer.
+
+It integrates the full functionality of gorc for operating as an orchestrator and relayer of [Gravity bridge](https://github.com/PeggyJV/gravity-bridge/) messages between the Ethereum and Cosmos chains.
+
+Stewards works in conjunction with the Orchestrator, so both processes must be running to fully participate in Cellar management.
+
+## Quickstart
+
+To start Steward, simply run
+
+```bash
+steward -c [path to your config toml] start
+```
+
+## Setting Up Steward
+
+In this section, let’s explore setting up steward for validators. There are two ways that validators will use Steward:
+
+1. Running Steward as a gRPC server for relaying Strategy Provider contract calls through the chain. If a validator is not running this server, it cannot participate in Cellar management.
+2. Manually scheduling contract calls when the validator set needs to coordinate administrative function calls on a cellar. An example of this would be shutting down a cellar.
+
+### Running Steward as a server
+
+Steward runs on every Validator in the Sommelier Validator set. It runs a server to which Strategy Providers (SPs) send requests whenever they determine that the market has changed enough to warrant action. The request payload contains everything needed to make a *cork*: a signed combination of a cellar address and an ABI encoded contract call. When Steward receives a submission from the SP, it validates the target cellar address, builds a cork, signs it with the delegate key, and submits it to the Cork module on chain.
```

Comment:
This is an extremely helpful overview of what Steward does. Breaking this information out and putting it in some overview document that other docs can link to would be great. (See comment above).